src/mod.ts

```diff
@@ -60,6 +60,13 @@ const startServer = async function (port: number) {
       } else if (
         requestEvent.request.method == "POST" && url.pathname == "/functions"
       ) {
+        // TODO: does the following need to be wrapped in a try/catch too?
+        // I think the await text() processes the read stream of the request
+        // Could probably blow up if the connection is interrupted
+        // TODO: I think we want to be intentional about how we use try/catch here.
+        // The in-line promise catch() would only catch issues responding to the
+        // request. The wrapping try{} would catch either userland function exceptions
+        // or JSON stringification of userland response.
```

Comment:
Don't think this needs addressing at this particular moment but as I was working here, I realized we may want to address some of these things at some point.